src/navsat_transform_nodelet.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2014, 2015, 2016, Charles River Analytics, Inc.
```

Comment:
Good point :-). This will take some time to clarify. Not sure which license I have to apply.